Source/Dafny/Compilers/Compiler-Csharp.cs

```diff
@@ -1479,7 +1479,7 @@ protected class ClassWriter : IClassWriter {
         if (Attributes.ContainsBool(cl.Attributes, "handle", ref isHandle) && isHandle) {
           return "0";
         } else {
-          return $"({TypeName(xType, wr, udt.tok)})null";
+          return $"(({TypeName(xType, wr, udt.tok)})null)";
```

Comment:
Do you expect this will ever part of a bigger expression?

Reply:
I changed it for consistency with `return $"(({TypeName(xType, wr, udt.tok)})null)";` above